src/locale/ar.ts

```diff
@@ -40,6 +41,9 @@ export default {
 
     [KEYS.CHAPTER_NEXT]: 'تالية سورة',
     [KEYS.CHAPTER_PREVIOUS]: 'السابقة سورة',
+    [KEYS.CHAPTER_VERSES]: 'AR VERSES',
```

Comment:
use google translate and use actual values, we can ask native Arabic speaker to fix any mistranslation later on,

Reply:
Sounds like something I can do.

Reply:
I will continue working on this over the weekend in sha Allah. Thank you for the comments, really appreciate it!